packages/lib-classifier/src/plugins/tasks/dropdown-simple/models/helpers/legacyDropdownAdapter/legacyDropdownAdapter.js

```diff
@@ -6,6 +6,7 @@ export function legacyDropdownAdapter(snapshot) {
     newSnapshot.allowCreate = menu.allowCreate
     newSnapshot.help = snapshot.help
     newSnapshot.instruction = snapshot.instruction
+    newSnapshot.required = snapshot.required
```

Comment:
It looks like the `required` property is stored in the individual `selects`. I think this line maybe should be `menu.required`, but I'm not entirely sure, will dig into more tomorrow.

Here's a screenshot from the linked workflow:
![Screen Shot 2022-09-13 at 6 14 54 PM](https://user-images.githubusercontent.com/12118751/190025688-1dee696d-9464-41d8-b402-0d3df85d69cd.png)

Reply:
Ohhhhhhh, that would explain it.

Reply:
I've updated the dropdowns in the tests to match your screenshot, and I've updated the code to pass the tests. It should work now.